### ISSUE TITLE
Weird issue with my weekly from mar-dec pickup schedule

### ISSUE THREAD

**Issue by GP920** [bruxy70/Garbage-Collection#59]
So with the latest update (version 2.9)... it seems that my once-a-week, from march - december pickup schedule is skipping all of 2020 and waiting for March 2021. 

My "yard waste" sensor says the next yard waste pickup is in 432 days... and not 60-something days that it should be. (March 4th, 2020 should be the next scheduled date)

Here's the section of my config (in case I screwed up the syntax or something):

```
  - name: "Yard Waste" # Each week on Wednesday, Between March and December
    frequency: "weekly"
    first_month: "mar"
    last_month: "dec"
    collection_days: "wed"
    icon_normal: mdi:leaf
    verbose_state: true
```

**Comment by bruxy70** [comment]
Thanks. Interesting. Looks like it did not realize that next week's Wednesday is already +1 year from today and it adds an extra year. Will look at that when I am back home next year.

**Comment by GP920** [comment]
No problem. Thanks!

**Comment by bruxy70** [comment]
It was actually a pretty simple fix. Fixed in 2.10

**Comment by GP920** [comment]
Awesome! Thanks!
